File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/plus_collection_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PlusCollectionVO:
    """Collection VO"""
    created_at: str = None
    updated_at: str = None
    id: int = None
    uuid: str = None
    name: str = None
    description: str = None
    type: str = None
    content_id: int = None
    cover_image: str = None
    is_public: bool = None
    is_pinned: bool = None
    sort: int = None
    item_count: int = None
    view_count: int = None
    favorite_count: int = None
    last_updated_at: str = None
    parent_id: int = None
    children_ids: List[int] = None

File: sdkwork-backend-sdk-go/types/plus_collection_vo.go
package types

// Collection VO
type PlusCollectionVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	Id int `json:"id"`
	Uuid string `json:"uuid"`
	Name string `json:"name"`
	Description string `json:"description"`
	Type string `json:"type"`
	ContentId int `json:"contentId"`
	CoverImage string `json:"coverImage"`
	IsPublic bool `json:"isPublic"`
	IsPinned bool `json:"isPinned"`
	Sort int `json:"sort"`
	ItemCount int `json:"itemCount"`
	ViewCount int `json:"viewCount"`
	FavoriteCount int `json:"favoriteCount"`
	LastUpdatedAt string `json:"lastUpdatedAt"`
	ParentId int `json:"parentId"`
	ChildrenIds []int `json:"childrenIds"`
}
